cpp/log/cert.h

```diff
@@ -272,6 +277,27 @@ class TbsCertificate {
   DISALLOW_COPY_AND_ASSIGN(TbsCertificate);
 };
 
+inline std::ostream& operator<<(std::ostream& out,
+                                const Cert::Status& status) {
+  switch (status) {
+    case Cert::Status::TRUE:
+      out << "TRUE";
+      return out;
+    case Cert::Status::FALSE:
+      out << "FALSE";
+      return out;
+    case Cert::Status::ERROR:
+      out << "ERROR";
+      return out;
+    case Cert::Status::UNSUPPORTED_ALGORITHM:
+      out << "UNSUPPORTED_ALGORITHM";
+      return out;
+  }
+
+  out << "<unknown Cert::Status: " << static_cast<int>(status) << ")";
```

Comment:
Surprised that you can cast an enum class value directly to int. But I'm often surprised by C++.